build.gradle

```diff
@@ -37,9 +37,10 @@ repositories {
 }
 
 dependencies {
-    compile ("uk.ac.ebi.subs:validator-common:0.3.0-SNAPSHOT")
+    compile ("uk.ac.ebi.subs:validator-common:0.4.0-SNAPSHOT")
```

Comment:
If you do any refactor above, then you can change it to the latest: 0.5.0-SNAPSHOT (currently)